### ISSUE TITLE
Umbilical discharge

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11317]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 34277280 Urachal abnormalities' incidence in children is 1 in 5000 and male to female ratio 3:1. Occurrence in adults is uncommon (2 per 100.000 hospitalizations) and constitutes a diagnostic challenge. Umbilical discharge/mass, abdominal pain, and hematuria are possible clinical presentations. **Umbilical discharge** aids in diagnosis. Pus discharge points towards urachal sinus. Urine discharge indicates patent urachus, and hematuria suggests vesico-urachal diverticulum.

**Parent term (use hpo.jax.org/app)**
Abnormality of the abdominal wall HP:0004298

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
urachal sinus ORPHA:431344
patent urachus ORPHA:431341
vesico-urachal diverticulum ORPHA:431347

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001416
- label: Umbilical discharge
- parents: [Abnormality of the abdominal wall (HP:0004298)](https://hpo.jax.org/app/browse/term/HP:0004298)
- pmid: PMID:34277280
- def: Abnormal fluid draining from the umbilicus (from wounds, surgical sites, or fistulas).
- comment: The dischanrge may be serous, purulent, bloody, or foul-smelling.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11317
- synonyms:
